CHANGELOG.md

```diff
@@ -5,11 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.2] - 2023-03-09
+### Fixed
+- Corrected an issue with the PyPI deployment for version 0.1.1. This version is the first available on PyPI.
 
-- Added new feature X
-- Fixed bug Y
-- Improved performance of Z
+## [0.1.1] - 2023-03-09
+### Added
+- Improved and expanded on documentation, including docstrings and READMEs.
+- Increased test coverage, especially with respect to the Problem and Solution classes.
 
-## [0.1.0] - 2023-02-xx
-- Initial release
\ No newline at end of file
+### Changed
+- Updated the MF Solver and DL Solver algorithms to better account for the enforced saturation value (S).
```

Comment:
@FarhadK-1QBit can you please help me better describe the changes that were made here, in these PRs?
#65 #69